style.css

```diff
@@ -0,0 +1,66 @@
+.menu {
+  display: flex;
+  justify-content: right;
+  padding: 20px;
+}
+
+.page {
+  display: flex;
+  flex-direction: column;
+  margin: unset;
+  overflow: auto;
+}
+
+.greetings {
+  justify-content: center;
+  font-size: 40px;
+  line-height: 52px;
+  margin-top: unset;
+  margin-bottom: unset;
+  color: #172b4d;
+}
+
+.job {
+  color: #36b37e;
+}
+
+.description {
+  font-weight: 400;
+  font-size: 20px;
+  line-height: 28px;
+  text-align: center;
+  color: #172b4d;
+}
```

Comment:
- [ ] Kindly make sure to use the correct typography (like `fonts-family`) on this paragraph in order to match the provided Figma design.

  - ### YOUR DESIGN
  
  ![image](https://github.com/KiengPhyl/Portfolio-setup-and-mobile-first1/assets/87197412/7fae001c-07c7-4df7-801d-afa572f41c28)


  - ### FIGMA DESIGN
  
    ![image](https://github.com/KiengPhyl/Portfolio-setup-and-mobile-first1/assets/87197412/40f591d5-a239-4d50-988c-d1bf44174f9f)